splash/browser_tab.py

```diff
@@ -672,7 +671,7 @@ def html(self):
         """ Return HTML of the current main frame """
         self.logger.log("getting HTML", min_level=2)
         frame = self.web_page.mainFrame()
-        result = bytes(frame.toHtml().encode('utf-8'))
+        result = frame.toHtml().encode('utf-8')
```

Comment:
Hmm .. as far as I understand, we feed this result into python2lua which ultimately converts unicode into bytes. So returning `frame.toHtml()` as is should be ok and more consistent.